### DISCUSSION TITLE
Basal Rates sent to Tidepool wrong / too high

### DISCUSSION THREAD

**Discussion by janvitos** [NightscoutFoundation/xDrip#2134] [Q&A]
**Subject of the issue**
xDrip is now sending the Basal Rates to Tidepool, but the values sent are often wrong and way too high.

**Your environment**
xDrip 2122051612
AndroidAPS 3.0.0.1
Dexcom G6
Android 12

**Expected behavior**
Basal Rates in Tidepool should always reflect the proper Basal Rates from xDrip and AndroidAPS.

**Actual behavior**
The current Basal Rate is 0.40 U/h, and because of the AndroidAPS closed loop and Temp Basals, it fluctuates between 0.00 U/h and 1.20 U/h. But in Tidepool, when the Basal Rate changes to 0.20 U/h, 0.50 U/h or 0.90 U/h for example, the Basal Rates shown in Tidepool are 20 U/h, 50 U/h and 90 U/h. This also results in a total daily Basal Insulin of 100 U+, when it should be closer to 10 U.

It seems like the Basal Rates are only good in Tidepool when they're at their base of 0.40 U/h. Otherwise, when the Temp Basal kicks in and Basal Rates fluctuate, xDrip (or Tidepool) seems to multiply the Basal by 10.

**Screenshots**
https://imgur.com/a/uqXRdnH

**Comment by Navid200** [comment]
@psonnera @jamorham

**Comment by Navid200** [comment]
https://github.com/NightscoutFoundation/xDrip/issues/2130
